File: week1/mario-less.c
#include <cs50.h>
#include <stdio.h>

void pyramid(int n);

int main(void)
{
    int height = 0;
   
    do
    {
        height = get_int("Height: ");
    }
    while (!(height >= 1 && height <= 8));
    
    pyramid(height);
}

void pyramid(int n)
{
    for (int i = 0; i < n; i++)
    {
        
        for (int k = n - i - 2; k >= 0; k--)
        {
            printf(" ");
        }        
       
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }        
   
        printf("\n");
    }
}

File: week 6/mario/mario-less.py
# TODO
from cs50 import get_int

height = 0
while (height > 8 or height < 1):
    height = get_int("Height:\n")

# prints a pyramid of hashes
for i in range(1, height + 1):
    print(" " * (height - i), end="")
    print("#" * (i))
